nixos/modules/virtualisation/oci-containers.nix

```diff
@@ -7,6 +7,53 @@ let
 
   defaultBackend = options.virtualisation.oci-containers.backend.default;
 
+  capabilities = [
```

Comment:
Is this list used anywhere?  I'm not seeing any references to it.

Reply:
It was supposed to be used to validate the cap add and cap drop options but it turns out it's not necessary since providing a nonexistent option will cause the service to fail to start. I'll remove it

Reply:
I removed that list